tests/unittests/evaluation_tests.py

```
@@ -3153,6 +3156,426 @@ def test_evaluate_video_segmentations_simple(self):
+
+        # Create temporary directory for test outputs
+        cls.temp_dir = tempfile.mkdtemp()
+
+    @classmethod
+    def tearDownClass(cls):
+        # Clean up the dataset
+        if hasattr(cls, "dataset"):
+            try:
+                cls.dataset.delete()
+            except:
+                pass
```

Ruff (0.8.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)